src/Microsoft.VisualStudio.Editor.Razor/DefaultTemplateEngineFactoryService.cs

```diff
@@ -7,10 +7,11 @@
 using Microsoft.AspNetCore.Razor.Language;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Razor;
 using Mvc1_X = Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X;
 using MvcLatest = Microsoft.AspNetCore.Mvc.Razor.Extensions;
 
-namespace Microsoft.CodeAnalysis.Razor
+namespace Microsoft.VisualStudio.Editor.Razor
 {
     internal class DefaultTemplateEngineFactoryService : RazorTemplateEngineFactoryService
```

Comment:
This guy and its base type could technically live in the Workspaces assembly but was torn on whether to move it or not. Therefore, less work was leaving it here.

Reply:
I think this is fine for now, this will eventually just be an implementation detail